client/my-sites/plans/jetpack-plans/product-store/hooks/use-included-product-description-map.tsx

```diff
@@ -64,7 +64,10 @@ export const useIncludedProductDescriptionMap = ( productSlug: string ) => {
 				translate( '1TB of ad-free video hosting.' )
 			),
 
-			...setTranslation( JETPACK_BOOST_PRODUCTS, translate( 'Automatic CSS generation.' ) ),
+			...setTranslation(
+				JETPACK_BOOST_PRODUCTS,
+				translate( 'Speed up your site and improve SEO with automatic Critical CSS generation.' )
```

Comment:
I'm not sure that the C in 'Critical' should be capitalized. Looking at our [landing page](https://jetpack.com/boost), it is not capitalized there

Reply:
You're right, @CodeyGuyDylan. I've just pushed a change to revise it.